app/forms/publication_date_form.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# Form object for handling publication date
+class PublicationDateForm < DateForm
+end
```

Comment:
We can revisit this decision later on. For now, I felt like having inherited forms was less noisy than adding an option to `ActiveModel::NestedAttributes` that isn't present in `ActiveRecord::NestedAttributes` (`form_class:`).